WordPress/Classes/ViewRelated/Post/WPLegacyEditPostViewController.m

```diff
@@ -390,7 +390,7 @@ - (void)cancelEditing
     }
 
     UIActionSheet *actionSheet;
-    if (![self.post.original.status isEqualToString:@"draft"] && self.editMode != EditPostViewControllerModeNewPost) {
+    if (![self.post.original.status isEqualToString:PostStatusDraft] && self.editMode != EditPostViewControllerModeNewPost) {
```

Comment:
Couldn't we create some helper methods in Post to check post status. Like isDraft, isPublished to avoid the repetition of direct string compares in the rest of the code?